<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomManageAssistantDlg</class>
 <widget class="QDialog" name="RoomManageAssistantDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RoomManageAssistantDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}
#wgt_title{
background:#2F2F2F;
}
#btn_forbiddenRecord,#btn_roomManage{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:#202020;
border:none;
}
#btn_forbiddenRecord::checked,#btn_roomManage:checked{
background:#262626;
color:#4AB134;
}
#page_forbiddenRecord,#page_roomManage{
background:#262626;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}
#btn_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#btn_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#btn_close:pressed{
margin:1px;
padding:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="text">
            <string>房管助手</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_forbiddenRecord">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>禁言记录</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_roomManage">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>房管管理</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">QTableView{
border:none;
background:#262626;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#777777;
}
QHeaderView{
background:transparent;
}
QHeaderView::section{
background:transparent;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#777777;
border-top:1px solid rgb(88, 87, 88);
border-bottom:1px solid rgb(88, 87, 88);
border-right:1px solid rgb(88, 87, 88);
border-radius:0px;
}
QHeaderView::section:first{
border-left:1px solid rgb(88, 87, 88);
}</string>
        </property>
        <widget class="QWidget" name="page_forbiddenRecord">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="ForbiddenRecordTableView" name="forbiddenRecordTableView">
            <property name="minimumSize">
             <size>
              <width>520</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>520</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_roomManage">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="RoomManageTableView" name="roomManageTableView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ForbiddenRecordTableView</class>
   <extends>QTableView</extends>
   <header>ForbiddenRecordTableView.h</header>
  </customwidget>
  <customwidget>
   <class>RoomManageTableView</class>
   <extends>QTableView</extends>
   <header>RoomManageTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
